package.json

```diff
@@ -2,10 +2,9 @@
   "name": "msw",
   "version": "2.7.6",
   "description": "Seamless REST/GraphQL API mocking library for browser and Node.js.",
-  "type": "commonjs",
-  "main": "./lib/core/index.js",
-  "module": "./lib/core/index.mjs",
-  "types": "./lib/core/index.d.ts",
+  "type": "module",
+  "main": "./lib/core/index.mjs",
```

Comment:
I heard the `main` field should be discouraged as it's effectively ignored by modern tooling.